README.md

```diff
@@ -1,2 +1,17 @@
 # deepcrawl
 A DeepCrawl API wrapper for Node
+
+```javascript
+var DeepCrawl = require('./');
+
+var deepCrawl = new DeepCrawl({
+  apiKey: '[your api key]',
+  apiId: '[your api id]',
```

Comment:
You'll probably want to default these to undefined so they'll throw in the constructor by default